### ISSUE TITLE
[Bug]: Classes rebuild command doesn't print classes

### ISSUE THREAD

**Issue by root913** [pimcore/pimcore#16325]
### Pimcore version

v11.1.2

### Steps to reproduce

Create any object class and execute classes rebuild command:
```bash
bin/console pimcore:deployment:classes-rebuild -vvv --create-classes --delete-classes
```

Line shown in screen below is causing this problem:
![image](https://github.com/pimcore/pimcore/assets/3902450/00a4485d-12a7-410a-a2f4-4b968806754d)

### Actual Behavior

Output of command:

```
---------------------
Saving all classes
---------------------
Saving all object bricks
---------------------
Saving all field collections
```

### Expected Behavior

```
---------------------
Saving all classes
Category [3] saved
Guarantee [4] saved
Producer [2] saved
Product [1] saved
Vat [5] saved
---------------------
Saving all object bricks
---------------------
Saving all field collections
```

**Comment by blankse** [comment]
@root913 I created a PR: https://github.com/pimcore/pimcore/pull/16326

**Comment by kingjia90** [comment]
Fixed by #16326
